packages/ui/src/app/pages/WorkingGroups/MyRoles/MyRole.tsx

```diff
@@ -48,104 +51,111 @@ export const MyRole = () => {
   }, [worker && worker.application.id])
   const isActive = worker && worker.status === 'WorkerStatusActive'
 
+  const [isLeaveModalVisible, toggleLeaveModal] = useToggle()
```

Comment:
I think we need a global modal:

1.  [https://www.figma.com/file/GlgN8uBRtvtMJtiOsdtDF7/Design?node-id=1740%3A86369](https://www.figma.com/file/GlgN8uBRtvtMJtiOsdtDF7/Design?node-id=1740%3A86369)
2.  https://www.figma.com/file/GlgN8uBRtvtMJtiOsdtDF7/Design?node-id=1740%3A92222